src/components/SectionsList/components/Section/index.jsx

```diff
@@ -21,6 +28,19 @@ const calcIcon = (type) => {
   }
 }
 
+const calcPDF = (short) => {
```

Comment:
Lekker! 😻

Reply:
Please let me know if there is a better way of doing this.